doc/rest.md

```diff
@@ -1,7 +1,7 @@
 # Docs for {`Rest`} client
 
 ```javascript
-import {Rest} from 'spoonx/aurelia-api';
+import {Rest} from 'aurelia-api';
```

Comment:
These changes already reside here: http://aurelia-api.spoonx.org/api_rest.html I think we can just remove the docs